serverless.yml

```diff
@@ -196,7 +188,7 @@ functions:
     handler: preferenceHandler.retrieveUserPreference
     events:
         - http:
-            path: api/preference/{id}
```

Comment:
Why is `{id}` removed from path, aren't we getting id from `pathParameters`?

Reply:
As this is user preference, I am fetching it via loggedIn user credential(cognitoIdentity).

Reply:
I see, that's good, since there is only one preference for the user.